### DISCUSSION TITLE
Is there a way to use PortIO (like inb and outb)

### DISCUSSION THREAD

**Discussion by CodeCs1** [CosmosOS/Cosmos#3040] [Q&A] [answered]
* Why ?
I found that there is no such function that perform some PortIO task.
* Question.
So is there a way to use that. If not , do I have to implement it by hand ?

**Comment by zarlo** [comment, marked as answer]
https://cosmosos.github.io/api/cosmos/Cosmos.Core.IOPort.html is that you are looking for

**Reply by CodeCs1** [reply]
yea, thanks
